tests/typescript/conformance/interfaces/interfaceDeclarations/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -44,8 +51,17 @@ interface Derived extends Base, Base2 {
   x: { b: string };
 }
 
-interface Derived2 extends Base, Base2 { // error
-  x: { a: number, b: string };
+// comment above TSInterfaceDeclaration
+interface Derived2 extends Base, Base2 // comment above TSInterfaceDeclaration
+{
+  // comment next TSInterfaceDeclaration
+  // comment above TSTypeLiteral
+  // comment inside of ObjectTypeAnnotation
+  // comment below TSTypeLiteral
```

Comment:
All comments get attached to the `TSInterfaceDeclaration` thus they all end up in one place. I think that used to work correctly.

@JamesHenry is this intentional? Then I would need to go through the whole body and use the location to attach the comments to the corresponding node before printing the body.